### PR TITLE
Added a twig extension to highlight some texts

### PR DESCRIPTION
**Pull request by ClementGautier**
| Q                 | A
| ----------------- | ---
| Specs             | Y
| Behats            | N/A
| Blue CI           | Y
| Changelog updated | Y
| Review and 2 GTM  | Y

In many screens we have to highlights some variables in texts and their is no easy way to do it actually.

### CODE REVIEW

File: src/Pim/Bundle/UIBundle/Twig/MiscExtension.php [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Pim\Bundle\UIBundle\Twig;
+
+/**
+ * Some presentation filters
+ *
+ * @author    Clement Gautier <clement.gautier@akeneo.com>
+ * @copyright 2015 Akeneo SAS (http://www.akeneo.com)
```
Comment (phaseinducer) [inline src/Pim/Bundle/UIBundle/Twig/MiscExtension.php]: 2016 :dancer:

File: spec/Pim/Bundle/UIBundle/Twig/MiscExtensionSpec.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+use Prophecy\Argument;
+
+class MiscExtensionSpec extends ObjectBehavior
+{
+    function it_is_a_twig_extension()
+    {
+        $this->shouldHaveType('\Twig_Extension');
+    }
+
+    function it_have_a_name()
```
Comment (phaseinducer) [inline spec/Pim/Bundle/UIBundle/Twig/MiscExtensionSpec.php]: has

File: spec/Pim/Bundle/UIBundle/Twig/MiscExtensionSpec.php [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    {
+        $this->getName()->shouldReturn('pim_ui_misc_extension');
+    }
+
+    function it_define_filters()
```
Comment (phaseinducer) [inline spec/Pim/Bundle/UIBundle/Twig/MiscExtensionSpec.php]: defines

File: CHANGELOG-1.5.md [hunk 4]
```diff
@@ -10,6 +10,7 @@
 - PIM-5217: Create a Buffer component and new file writer implementations that use it
 - PIM-4646: TinyMCE wysiwyg editor is replaced by Summernote in the mass-edit and variant group
 - PIM-4999: jQuery UI datepicker is replaced by bootstrap datepicker in the mass-edit and variant group
+- A MiscExtension in UIBundle now provides a "highlight" string filter
```
Comment (jmleroux) [inline CHANGELOG-1.5.md]: No linked ticket ?

**Comment by phaseinducer** [comment]
GTM after fix

File: src/Pim/Bundle/UIBundle/Resources/config/twig.yml [hunk 5]
```diff
@@ -0,0 +1,9 @@
+parameters:
+    pim_ui.twig.misc_extension.class: Pim\Bundle\UIBundle\Twig\MiscExtension
+
+services:
+    pim_ui.twig.misc_extension:
+        class: %pim_ui.twig.misc_extension.class%
+        public: false
```
Comment (skeleton) [inline src/Pim/Bundle/UIBundle/Resources/config/twig.yml]: :+1:

File: CHANGELOG-1.5.md (hunk 4 above)
Comment (juliensnz) [inline CHANGELOG-1.5.md]: please create a ticket

File: src/Pim/Bundle/UIBundle/Twig/MiscExtension.php [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class MiscExtension extends \Twig_Extension
```
Comment (juliensnz) [inline src/Pim/Bundle/UIBundle/Twig/MiscExtension.php]: No way :)

Comment (jmleroux) [inline src/Pim/Bundle/UIBundle/Twig/MiscExtension.php]: `StyleExtension` could be better

**Comment by skeleton** [comment]
GTM once MiscExtension has been renamed :)

**Comment by jmleroux** [comment]
GTM after better naming

File: CHANGELOG-1.5.md (hunk 4 above)
Comment (ClementGautier) [inline CHANGELOG-1.5.md]: Nope, I couls have done that without the extension but I though it would be useful for all ;)

**Pull request merged**